src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AwsSdk/AwsSdkPipelineWrapper.cs

```diff
@@ -115,6 +115,7 @@ public AfterWrappedMethodDelegate BeforeWrappedMethod(InstrumentedMethodCall ins
             // modify the request to ask for DT headers in the response message attributes
             if (action == MessageBrokerAction.Consume)
             {
+                // TODO: update the distributed trace API to give the set of headers to accept based on current configuration
```

Comment:
Didn't mean to include this in my last commit. It's a reminder for the next PR that I'm working on...